c++/src/kj/string.h

```diff
@@ -600,40 +725,72 @@ inline ArrayPtr<const char> StringPtr::slice(size_t start, size_t end) const {
   return content.slice(start, end);
 }
 
+inline LiteralStringConst::operator ConstString() const {
+  return ConstString(begin(), size(), NullArrayDisposer::instance);
+}
+
+inline ConstString StringPtr::attach() const {
+  // This is meant as a roundabout way to make a ConstString from a StringPtr
+  return ConstString(cStr(), size(), NullArrayDisposer::instance);
```

Comment:
You missed a spot.

Reply:
I do see that my own code elsewhere does the same thing though, so maybe I'm being too nitpicky.